packages/cashier/src/components/recent-transaction/__tests__/recent-transaction.spec.tsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import { fireEvent, render, screen } from '@testing-library/react';
 import RecentTransaction from '../recent-transaction';
-import { createBrowserHistory } from 'history';
+import { BrowserHistory, createBrowserHistory } from 'history';
 import { Router } from 'react-router';
 import CashierProviders from '../../../cashier-providers';
+import { mockStore } from '@deriv/stores';
+import { TRootStore } from '@deriv/stores/types';
 
 describe('<RecentTransaction />', () => {
-    let history, mockRootStore;
+    let history: BrowserHistory, mockRootStore: TRootStore;
```

Comment:
The type of the `mockRootStore` should be `TStores` from `@deriv/store`.

```suggestion
    let history: BrowserHistory, mockRootStore: TStores;
```